sections/main-product.liquid

```diff
@@ -106,6 +106,51 @@
                   {{ form | payment_terms }}
                 {%- endform -%}
               </div>
+            {%- when 'inventory' -%}
+              <p
+                class="product__inventory no-js-hidden{% if block.settings.text_style == 'uppercase' %} caption-with-letter-spacing{% elsif block.settings.text_style == 'subtitle' %} subtitle{% endif %}{% if product.selected_or_first_available_variant.inventory_management != 'shopify'%} visibility-hidden{% endif %}"
+                {{ block.shopify_attributes }}
+                id="Inventory-{{ section.id }}"
+                role="status"
```

Comment:
I wonder if we should actually apply this role in all situation 🤔 The reason I say that is because we're already conveying the sold out and unavailable state when selecting different variant options. So it doubles the announcement about it. 
Maybe we can conditionally apply it when the inventory quantity is above `0` or `when it's 0 and inventory_policy == 'continue'`.

Reply:
For this I have created a follow up issue for us to find a better solution to announcing the information without repeating ourselves too much: https://github.com/Shopify/dawn/issues/2061